.github/workflows/kani.yml

```diff
@@ -192,6 +192,12 @@ jobs:
     container:
       image: ubuntu:18.04
     steps:
+      - name: Remove unnecessary software to free up disk space
+        run: |
+          # inspired by https://github.com/easimon/maximize-build-space/blob/master/action.yml
+          df -h
+          rm -rf /usr/share/dotnet /usr/local/lib/* /opt/*
```

Comment:
Isn't this a bit too aggressive and could result in deleting binaries we need? Should we make it a bit more specific, e.g. `/usr/local/lib/android` and `/opt/ghc`?

Reply:
We can also using an action: https://github.com/marketplace/actions/maximize-build-disk-space but this might be an overkill.

Reply:
I think if we're deleting binaries that we need then we'll notice in subsequent CI steps? I opted for the immediate commands over the existing action for improved transparency. The commands themselves are inferred from that action's code as indicated in my comment.